cmake/modules/RootBuildOptions.cmake

```diff
@@ -402,8 +402,7 @@ endforeach()
 
 #---Replaced options--------------------------------------------------------------------------
 if(python)
-  message(DEPRECATION ">>> Please use the equivalent option 'pyroot' instead of 'python'; the latter will be removed in v6.22.")
-  set(pyroot ON CACHE BOOL "" FORCE)
+  message(STATUS ">>> INFO: 'python' option was removed. Instead, it was enabled a 'pyroot' option.")
```

Comment:
Perhaps `Instead, the 'pyroot' option should be used.` ?

`it was enabled` can be understood as "we enabled the pyroot option for you instead", which we don't.

Reply:
@etejedor, pyroot is always enabled (default=ON), so my logic was that 'python' option is not valid, and pyroot option is already enabled.

Reply:
I changed text a bit and  I need approval to merge it, thanks!